docs/data/material/components/buttons/buttons.md

```diff
@@ -116,6 +116,12 @@ Use `color` prop to apply theme color palette to component.
 
 {{"demo": "IconButtonColors.js"}}
 
+:::info
+Note: You can only use 8 colors with color attribute `inherit | default | primary | secondary | error | info | success | warning`, which gives limited usability of colors. To bring more customization to color we can use `style` attribute.
+:::
```

Comment:
```suggestion
To extend the color customization capacity, use the `style` prop with the `color` attribute.
```

Thanks for pushing these last changes! From reading it in context, again, I feel like removing it from the info callout makes sense. It's delivering key info. Also suggesting a bit of a simplified version of it.